### PR TITLE
Left panel update

### PR DESCRIPTION
**Pull request by june-tran**

### CODE REVIEW

File: components/ui/left-panel/leftPanel.tsx [hunk 1]
```diff
@@ -72,54 +70,63 @@ export function LeftPanel(props: LeftPanelProps ) {
...
-          <div className="flex justify-between w-full items-center"> 
-            {sceneTitle} 
-
-            <button className="p-1 rounded w-8 h-8 hover:bg-blue-500 "> 
+          <div className="flex justify-between w-full"> 
+            <div 
+              id = "editableTitle"
+              className=" max-w-[80%] mr-2 break-all" 
+              spellCheck="false"
+              // TODO: retrieve object so that its name can be updated
```
Comment (notalfredo) [inline components/ui/left-panel/leftPanel.tsx:85]: This has already been done right ?

Comment (june-tran) [reply components/ui/left-panel/leftPanel.tsx:85]: Oh yes. I forgot to remove the comment

Comment (notalfredo) [reply components/ui/left-panel/leftPanel.tsx:85]: Can you remove it ?

**Review by notalfredo — APPROVED** [review]
im going to approve the PR. When you fix the issue just merge it your self

**Pull request merged**